basics/10_file_handling/2_create_file_folder.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+echo "<pre>";
+
+echo "<h1>Membuat Folder dan File dengan PHP</h1>";
+
+echo "Untuk membuat folder pada PHP gunakan mkdir()
+
+// Menyimpan data untuk membuat Folder ke dalam Variabel
+dir = 'storage/contoh_folder';
+
+// Identitas File yang akan dibuat
+nama_file = 'test.txt';
+konten_file = 'contoh content file';
+
+// Membuat Folder
+mkdir(dir);
+
+
+// fopen sebelum membuat file
+open = fopen(dir . '/' . nama_file, 'w');
+
+// fwrite untuk membuat sebuah file dan kontennya
+fwrite(open, konten_file);
+
+// Menutup File
+fclose(open);
+
+<b>Hasil nya dapat dilihat setelah halaman ini di load silakan cek folder storage seharusnya sudah ada folder dan juga file baru di dalamnya:</b>
+
+";
+
+// Menyimpan data untuk membuat Folder ke dalam Variabel
+$dir = 'storage';
+
+// Identitas File yang akan dibuat
+$nama_file = 'test.txt';
+$konten_file = 'contoh content file';
+
+// Membuat Folder
+mkdir($dir);
```

Comment:
sedikit request nih, saat file demo dijalankan pertama kali, dan dijalankan lagi muncul error seperti ini
![image](https://user-images.githubusercontent.com/13761315/155349809-e3890eae-ded6-4405-b42f-44f141ef8b79.png)
Jadi ada baiknya jika dikasih validation seperti ini untuk mencegah errornya
```suggestion
// Mengecek apakah folder storage belum ada
if (!is_dir('./storage')) {
    // Membuat Folder jika belum ada
    mkdir($dir);
}
```